protocol/src/main/java/org/apache/drill/exec/proto/beans/CoreOperatorType.java

```diff
@@ -78,7 +78,8 @@
     SEQUENCE_SUB_SCAN(53),
     PARTITION_LIMIT(54),
     PCAPNG_SUB_SCAN(55),
-    RUNTIME_FILTER(56);
+    RUNTIME_FILTER(56),
+    ROWKEY_JOIN(57);
```

Comment:
Please regenerate protobuf files for C++ native client as well

Reply:
I have regenerated the protobuf files for the native C++ client.  (As an aside, the steps for doing this seems complicated unlike the Java version. In either case, we really should not need to rebuild the client side if a new operator has been added on the server side).